code/lib/cli/src/sandbox-templates.ts

```diff
@@ -347,7 +347,8 @@ const baseTemplates = {
   },
   'vue-cli/default-js': {
     name: 'Vue-CLI (Default JS)',
-    script: 'npx -p @vue/cli vue create . --default --packageManager=yarn --force --merge',
+    script:
+      'npx -p @vue/cli vue create . --default --packageManager=yarn --force --merge && echo "module.exports = {}" > webpack.config.js',
```

Comment:
is there any side effect of having an empty webpack config? like will storybook use it instead of its defaults? I assume it will merge them so it should be fine, but did you check if initializing Storybook and running them in this sandbox works as expected?

Reply:
Just checking it right now

Reply:
LGTM. No side-effects